File: jknowsbackend/src/main/java/com/jknows_backend/service/AnswerService.java
package com.jknows_backend.service;

import com.jknows_backend.entity.Answer;

import java.util.List;


public interface AnswerService {

    List<Answer> getAnswer(int questionId);

    List<Answer> getAnswers(int questionId, int start, int end);

    List<Answer> getUserAnswer(int userId);

    Answer saveAnswer(int answerId, int userId, int questionId, String content);

    int deleteAnswer(int answerId);

    Answer getMyAnswer(int questionId, int userId);
}

File: jknowsfrontend/src/service/answerService.js
import {postRequest,getRequest} from "../utils/ajax";
export const getUserAnswer = (userId,callback) => {
    const data = {userId : userId};
    const url = "http://123.60.52.71:8081/getUserAnswer"
    postRequest(url, data, callback);
};
export const saveAnswer=(answerId, userId, questionId, content,callback)=>{
    const data = {answerId:answerId,userId:userId,questionId:questionId,content:content};
    const url = "http://123.60.52.71:8081/saveAnswer"
    postRequest(url, data, callback);
}
export const getLikeAnswer = (userId,callback) => {
    const data = {userId : userId};
    const url = "http://123.60.52.71:8081/getLikeAnswer"
    postRequest(url, data, callback);
};